### ISSUE TITLE
[Bug]: Component Tag colors does not have expected hex value

### ISSUE THREAD

**Issue by rohitrajmishra** [carbon-design-system/carbon#17629]
### Package

@carbon/themes

### Package version

v11.39.0

### React version

v16.14.0

### Description

As per color guide/documentation 

https://carbondesignsystem.com/elements/color/tokens/#tag

the color for token `$tag-color-magenta` we expect color `magenta70 -- #9f1853`,

But in practice the resulting color is `magenta80  -- #740937`.

Similar is the case with other tag colors also.
E.g.
```
$tag-color-purple,
$tag-color-blue,

...
and so on
```

This results in darker shade of colors applied to app then design intended.

### Reproduction/example

https://stackblitz.com/edit/github-sxced5-yboywq?file=src%2Fmain.jsx

### Steps to reproduce

1. Setup
Scss/Css code snippet to reproduce the issue,

```
@use '@carbon/styles/scss/components/tag';

<your-css-selector> {

color: tag.$tag-color-magenta;

}
```

2. Compare resulting color(`magenta80` with what is mentioned in `white/g10` theme(`magenta70`)
https://carbondesignsystem.com/elements/color/tokens/#tag

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by rohitrajmishra** [comment]
<img width="1792" alt="Screenshot 2024-10-02 at 2 04 25 AM" src="https://github.com/user-attachments/assets/6cfd7d55-427f-4cd4-bdde-ae57d064e16a">

**Comment by rohitrajmishra** [comment]
We tried to investigate a bit,
It seems below is suspected code in repo,

https://github.com/carbon-design-system/carbon/blob/7e6ec8e89df9caf3f27e167cdf8be42783e93682/packages/themes/src/component-tokens/tag/tokens.js#L105

Code in this file mentions `magenta80`and `80` shade for other colors for `white/g10` theme, which is different from documentation which mentions `magenta70` or `70` shade for such color tokens.

**Comment by aagonzales** [comment]
Yes, I am seeing the same thing. I've found this PR https://github.com/carbon-design-system/carbon/pull/10287 where we updated the dark theme tag tokens to the 70 scale but not the light theme ones. I'm trying to figure out if we've since decided to update the light values as well and which assets have the bug. 

![Image](https://github.com/user-attachments/assets/d120133f-9af0-427f-8672-de902ef8b9bf)
